dspace/config/modules/info-actuator.cfg

```diff
@@ -0,0 +1,19 @@
+
+info.app.name = ${dspace.name}
+info.app.version = 7.3
```

Comment:
We should avoid hardcoding a version here in configuration as this is not accurate.  We already report a programmatically determined version in `dspaceVersion` property from `/server/api`.  So, if this setting cannot be applied programmatically, then we should just remove `info.app.version` altogether...as it'll likely be wrong.